Test/Calculator.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../src/Components/Calculator';
+
+describe('Calculator component', () => {
+  it('renders correctly', () => {
+    const component = renderer.create(<Calculator />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] One of the requirements is to use React Testing Library to simulate user interaction in the application but it has not been executed. Kindly implement this requirement. You can learn more about React Testing Library by reading these resources:
- [Documentation](https://testing-library.com/docs/react-testing-library/intro/)
- [External Resource](https://www.robinwieruch.de/react-testing-library/) 💻 🍪

Reply:
This change has been done well